plugins/experimental/wasm/wasm_main.cc

```diff
@@ -46,6 +46,249 @@ struct WasmInstanceConfig {
 
 static std::unique_ptr<WasmInstanceConfig> wasm_config = nullptr;
 
+// handler for transform event
+static int
+transform_handler(TSCont contp, ats_wasm::TransformInfo *ti, TSEvent event)
```

Comment:
TSEvent event not needed in transform_handler

Reply:
let me remove this